go/libraries/doltcore/sqle/writer/prolly_table_writer.go

```diff
@@ -324,9 +325,6 @@ func (w *prollyTableWriter) table(ctx context.Context) (t *doltdb.Table, err err
 }
 
 func (w *prollyTableWriter) flush(ctx *sql.Context) error {
-	if !w.primary.HasEdits(ctx) {
```

Comment:
I noticed that we call `flush` multiple times when we should only be calling it once. Was this causing errors?

Reply:
That's a good point. It's worth adding a comment.